src/main/cpp/chassis/states/TrajectoryDrive.h

```diff
@@ -67,13 +65,11 @@ class TrajectoryDrive : public RobotDrive
     std::string m_whyDone;
     units::time::second_t m_totalTrajectoryTime;
 
-    double m_kPCoarse = 5.0;
-    double m_kPFine = 9.0;
-    const double m_percentageCompleteThreshold = 0.90;
-    int m_samePoseCount = 0;
-    const int m_samePoseCountThreshold = 50; // TODO come back and tune this
+    frc::PIDController m_xController{0.35, 0.0, 0.0};
```

Comment:
In general it is nice to have constants that are pulled out of the creation so inexperienced people understand what the values being passed are.   Some may even be able to be re-used (e.g. noIntegralCoefficent).

Reply:
Done

Reply:
Connector is better than const

Reply:
It wouldn't let me make them a constexpr for some reason

Reply:
It might be because you are using the = initializer.  It is actually preferred to use the {} initializer as it removes copy/move logic and does better type safety